dependencies/pytest-min/constraints.txt

```diff
@@ -14,7 +14,7 @@ packaging==23.2
 pluggy==1.3.0
 py==1.11.0
 Pygments==2.16.1
-pytest==7.0.0
+pytest==8.0.0
```

Comment:
The intention of the `pytest-min` environment is to test against an older version of pytest. This way, pytest-asyncio doesn't accidentally use any of the newer pytest features without a proper bump of the minimum supported version.

I think we should continue to support pytest 7 for at least a short period of time, in order to allow users to migrate pytest independently from pytest-asyncio.